policy_set.go

```diff
@@ -71,9 +71,18 @@ type PolicySet struct {
 	UpdatedAt      time.Time `jsonapi:"attr,updated-at,iso8601"`
 
 	// Relations
+	// The organization to which the policy set belongs to.
 	Organization *Organization `jsonapi:"relation,organization"`
-	Policies     []*Policy     `jsonapi:"relation,policies"`
-	Workspaces   []*Workspace  `jsonapi:"relation,workspaces"`
+	// The workspaces to which the policy set applies.
+	Workspaces []*Workspace `jsonapi:"relation,workspaces"`
+	// Individually managed policies which are associated with the policy set.
+	Policies []*Policy `jsonapi:"relation,policies"`
+	// The most recently created policy set version, regardless of status.
+	// Note that this relationship may include an errored and unusable version,
+	// and is intended to allow checking for errors.
+	PolicySetVersionNewest *PolicySetVersion `jsonapi:"relation,newest-version"`
+	// The most recent successful policy set version.
+	PolicySetVersionCurrent *PolicySetVersion `jsonapi:"relation,current-version"`
```

Comment:
How about `NewestVersion` and `CurrentVersion`?

The context of it being a version of a policy set is present in the owning type (PolicySet); that's why the attributes themselves are called `newest-version` and `current-version`; this would also match `Workspace.CurrentRun` (rather than `RunCurrent`!)

Reply:
Added